Update readme (#96)
DigitalSlideArchive/ImageDePHI: README.md

# ImageDePHI

> ImageDePHI is an application that can be used to redact PHI from whole slide images.

Reviewer: ```suggestion
ImageDePHI is an application to redact PHI from whole slide images.
```

This project has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, Department of Health and Human Services, under Contract No. 75N91022C00033

# Installation

> ImageDePHI is under active development and as such the best way to test the latest features is to download the most current artifact.

Reviewer: https://github.com/DigitalSlideArchive/ImageDePHI/releases/latest will take you to the latest release

# ImageDePHI
ImageDePHI is an application that can be used to redact PHI from whole slide images.

# Installation
ImageDePHI is under active development and as such the best way to test the latest features is to download the most current artifact.

[ImageDePHI Artifacts](https://github.com/DigitalSlideArchive/ImageDePHI/actions/workflows/ci.yml?query=branch%3Amain)

Select the most recent run:
![ci run](imagedephi/assets/ci_run.png)

Then select the appropriate artifact for your OS:

> ![artifacts](imagedephi/assets/artifacts_list.png)

Reviewer: I think we can have a change very soon which just publishes binaries directly with release. Asking people to go into CI runs is too burdensome just to download an application.

Author: Once we have a release I will update this to link to that instead

# Usage
## CLI:
Open a terminal and navigate to the directory where the artifact was downloaded.

run:
```
./imagedephi
```
to get a list of cli options

```
Usage: imagedephi [OPTIONS] COMMAND [ARGS]...

  Redact microscopy whole slide images.

Options:
  -r, --override-rules FILENAME  Specify user-defined rules to override
                                 defaults
  --help                         Show this message and exit.

Commands:
  gui   Run a web-based GUI.
  plan  Print the redaction plan for a given image and rules.
  run   Redact images in a folder according to given rule sets.
```

To redact a set of images supply the directory of the images to be processed and [optionally] an output directory to deposit the redacted images:
```
imagedephi run [path/to/images] [optional/output/path]

> ```

Reviewer: I don't think we should include this output. It's going to be changing frequently, it may be slightly different on different platforms (e.g. #75), and it will be burdensome for us to keep these docs up to date.

Author: removed in a89e9c9d71a8b2ec6a0ddaa81f819ffd05d1f6e7

## GUI

> Alternatively you can use the gui to navigate to the desired directories:

Reviewer: ```suggestion
Alternatively you can use the GUI to navigate to the desired directories:
```

Author: fixed in a89e9c9d71a8b2ec6a0ddaa81f819ffd05d1f6e7

```
imagedephi gui
```

> ### **Note**: Windows users can double click the imagdephi executable icon to launch the gui outside of the terminal workflow

Reviewer: ```suggestion
### **Note**: To launch the ImageDePHI GUI, Windows users can directly run `imagdephi.exe` from Windows Explorer.
```

Author: fixed in a89e9c9d71a8b2ec6a0ddaa81f819ffd05d1f6e7
